packages/terra-dropdown-button/src/terra-dev-site/test/dropdown-button/DefaultDropdownButton.test.jsx

```diff
@@ -8,9 +8,9 @@ const cx = classnames.bind(styles);
 export default () => (
   <div className={cx('container-spacing-wrapper')}>
     <DropdownButton label="Dropdown" id="dropdown">
-      <Item label="1st" onSelect={() => {}} />
-      <Item label="2nd" onSelect={() => {}} />
-      <Item label="3rd" onSelect={() => {}} />
+      <Item label="1st" metaData={{ key: '1st Option' }} onSelect={(event, metaData) => { console.log('Metdata of :', metaData.key); }} />
+      <Item label="2nd" metaData={{ key: '2nd Option' }} onSelect={(event, metaData) => { console.log('Metdata of :', metaData.key); }} />
```

Comment:
Logs do not really help validate the functionality. I think you could enhance this to maybe enclose this in a `<p>` tag or something and show it on the page so that this can be caught in the test screenshot when either the primary button or an item is selected.

Reply:
Added <p> to display selected value : [code link](https://github.com/cerner/terra-core/blob/abe8f36b8488383183a4a06827b136a3c0b83a45/packages/terra-dropdown-button/src/terra-dev-site/test/dropdown-button/DefaultDropdownButton.test.jsx#L13-L16)